sass/_Omnibox.scss

```diff
@@ -18,6 +18,8 @@
     color: $black;
   }
   .magic-box-underlay {
+    display: flex;
```

Comment:
Does it still look good if you enable the query syntax on the search box, and start typing `@objecttype=` ?